spec/PlayerSpec.js

```diff
@@ -0,0 +1,58 @@
+describe("Player", function() {
```

Comment:
I am going to presume that this is the left over jasmine scripts from the standalone files hahaha.